tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "declaration": true,
     "outDir": "dist",
     "strict": true,
+    "skipLibCheck": true,
```

Comment:
@mojosoeun  I have a question..! Is there any reason to skip d.ts type checking??

Reply:
After I upgrade the typedoc version, `--ignoreCompilerErrors` was deprecated. when I generate docs, I got Lib('grpc-web') compile error. So I added skipLibCheck typescript config

Reply:
After I upgrade the typedoc version, `--ignoreCompilerErrors` was deprecated. when I generate docs, I got Lib('grpc-web') compile error. So I added skipLibCheck typescript config